tool/tsh/db.go

```diff
@@ -1001,8 +1016,11 @@ func formatDatabaseConfigCommand(clusterFlag string, db tlsca.RouteToDatabase) s
 // for connecting to the provided database. Currently return true if:
 //   - TLS routing is enabled.
 //   - A SQL Server connection always requires a local proxy.
+//   - A Cassandra connection always requires a local proxy.
 func isLocalProxyRequiredForDatabase(tc *client.TeleportClient, db *tlsca.RouteToDatabase) bool {
-	return tc.TLSRoutingEnabled || db.Protocol == defaults.ProtocolSQLServer
+	return tc.TLSRoutingEnabled ||
+		db.Protocol == defaults.ProtocolSQLServer ||
+		db.Protocol == defaults.ProtocolCassandra
```

Comment:
Why Cassandra requires a local proxy?

Reply:
Since the Teleport Connect always uses the local proxy with `--tunnel` flag we want to unify the access for new protocol and improve the UX. Also we have implemented feature in local proxy base on TLS Routing like artificial database ping protocol that are leverage by local proxy.